modules/acquia_cms_image/acquia_cms_image.install

```diff
@@ -5,15 +5,53 @@
  * Install, update and uninstall functions for the acquia_cms_image module.
  */
 
+use Drupal\Core\File\FileSystemInterface;
 use Drupal\Core\Installer\InstallerKernel;
 use Drupal\image\Entity\ImageStyle;
+use Drupal\media\Entity\Media;
 
 /**
  * Implements hook_install().
  */
 function acquia_cms_image_install($is_syncing) {
   if (!$is_syncing) {
     _acquia_cms_common_editor_config_rewrite(TRUE);
+
+    // Create file entity.
+    $acms_image_path = \Drupal::moduleHandler()->getModule('acquia_cms_image')->getPath();
+    $image_data = file_get_contents($acms_image_path . '/assets/images/acquia_cms_logo.png');
+    $file_repository = \Drupal::service('file.repository');
+    $acquia_cms_logo_uri = "public://media-icons/acquia_cms_logo.png";
+    $image = $file_repository->writeData($image_data, $acquia_cms_logo_uri, FileSystemInterface::EXISTS_REPLACE);
+    $image->setFileName('Acquia CMS logo');
+    $image->setMimeType('image/png');
+    $image->setPermanent();
+    $image->save();
+
+    // Create media for Acquia CMS logo.
+    $acquia_cms_logo = Media::create([
+      'name' => 'Acquia CMS Logo',
+      'bundle' => 'image',
+      'uuid' => '0c6f0f26-9fbb-4c2e-804c-418815aba162',
+      'image' => [
+        'target_id' => $image->id(),
+        'alt' => t('Acquia CMS logo'),
+        'title' => t('Acquia CMS logo'),
+        'width' => 286,
```

Comment:
@rajeshreeputra Let's not hardcode height/width here. I think we can get height/width from `$image` object.

Reply:
Also, please throw exceptions i.e
- File `$acms_image_path . '/assets/images/acquia_cms_logo.png'` must exist and readable.
- The public path is writable.

Reply:
done